stats/src/main/java/com/facebook/stats/mx/JVMStatsExporter.java

```diff
@@ -100,12 +171,14 @@ public static Stats createAndBindTo(Stats stats) throws JMException {
    * them as well.
    * 
    * @param beanNamePattern the bean name pattern used to discover the beans.
+   * @param statsNameBuilder the bean name pattern used to discover the beans.
    * @see javax.management.ObjectName for bean name pattern syntax
    * @see java.lang.management for the list of java platform mbeans
    * 
    * @throws JMException if there are errors querying MBeans or information on them
    */
-  public void exportNumericAttributes(ObjectName beanNamePattern) throws JMException {
+  public void exportNumericAttributes(ObjectName beanNamePattern, StatsNameBuilder statsNameBuilder)
```

Comment:
**This breaks backwards compatibility!** We should **not** change this method signature (see comment above in constructor about how to avoid needing to).